File: 07_backend/01_MiProyecto/src/main/java/modelo/Persona.java
package modelo;

public class Persona implements java.io.Serializable {
	private String nombre;
	private String cedula;
	private String apellido;
	
	public Persona() {}
	
	

	public Persona(String nombre, String cedula, String apellido) {
		super();
		this.nombre = nombre;
		this.cedula = cedula;
		this.apellido = apellido;
	}



	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getCedula() {
		return cedula;
	}

	public void setCedula(String cedula) {
		this.cedula = cedula;
	}

	public String getApellido() {
		return apellido;
	}

	public void setApellido(String apellido) {
		this.apellido = apellido;
	}

	@Override
	public String toString() {
		return "Persona [nombre=" + nombre + ", cedula=" + cedula + ", apellido=" + apellido + "]";
	}

	
	
}

File: iniciophp/07_ConexionBDD/Persona.php
<?php
/**
 * Clase que representa una persona.
 */
class Persona {
	private $id;
	private $nombre;
	private $apellido;
	private $edad;

	/**
	 * Constructor vacío o con atributos excepto id.
	 * En PHP no se pueden definir dos constructores con distinta firma,
	 * por eso se usan parámetros opcionales.
	 */
	public function __construct($nombre = null, $apellido = null, $edad = null) {
		$this->nombre = $nombre;
		$this->apellido = $apellido;
		$this->edad = $edad;
	}

	public function getId() {
		return $this->id;
	}

	public function setId($id) {
		$this->id = $id;
	}

	public function getNombre() {
		return $this->nombre;
	}

	public function setNombre($nombre) {
		$this->nombre = $nombre;
	}

	public function getApellido() {
		return $this->apellido;
	}

	public function setApellido($apellido) {
		$this->apellido = $apellido;
	}

	public function getEdad() {
		return $this->edad;
	}

	public function setEdad($edad) {
		$this->edad = $edad;
	}
}
?>
